### PR TITLE
Add jupyterlab-variableinspector

### PR DESCRIPTION
**Pull request by fcollonval**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] ~~If static libraries are linked in, the license of the static library is packaged.~~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by fcollonval** [comment]
@conda-forge/staged-recipes this is ready for review

File: recipes/jupyterlab-variableinspector/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f80a742b7e56c613afd6759130963a9d574daecbd092014263e61303049e2839
+
+build:
+  noarch: python
+  number: 0
+  script: '{{ PYTHON }} -m pip install . --install-option="--skip-npm" -vv'
```
Comment (ocefpaf) [inline recipes/jupyterlab-variableinspector/meta.yaml:15]: @bollwyvl do you mind taking a quick look at this one before we merge?

Comment (fcollonval) [reply recipes/jupyterlab-variableinspector/meta.yaml:15]: Thanks @ocefpaf  for the review

I used that option following a suggestion of @bollwyvl in that conda recipe : https://github.com/conda-forge/staged-recipes/pull/14182#discussion_r588899830 😉

Comment (ocefpaf) [reply recipes/jupyterlab-variableinspector/meta.yaml:15]: OK. I'm going to use that as a @bollwyvl seal of approval ;-p

**Pull request merged**